test/integration-tests/src/test/resources/email.properties

```diff
@@ -0,0 +1,6 @@
+mail.smtp.host=smtp.gmail.com
```

Comment:
We are not reading these currently!

Reply:
This is bring used in the mail session object